tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -32,6 +32,8 @@
 
   <ItemGroup>
     <None Remove="AppSec\Waf\rule-set.json" />
+    <None Remove="ClrProfiler\ServerlessInstrumentation\" />
+    <None Remove="ClrProfiler\ServerlessInstrumentation\AWS\" />
```

Comment:
You can revert this change and the `<Folder Include=...>` lines below. These are removing the folders from the build, and lines below are undoing that by adding them back again.